src/test/java/MapersTests/MapCharacterTests.java

```diff
@@ -0,0 +1,36 @@
+    private final MapCharacterClass testObject = new MapCharacterClass();
+    private final String badOutputMapper = "{\"ability_score\":{\"index\":\"dex\",\"name\":\"DEX\",\"url\":\"/api/ability-scores/dex\"},\"noslda\":2}";
+
+    @Test
+    public void mapperHappyPath() {
+        String resultForString = testObject.mapCharacterClass(outputResponseBody).toString();
+        Assertions.assertEquals(outputMapper, resultForString);
+    }
+
+    @Test
+    public void mapperUNHappyPath() {
+        Assertions.assertNotEquals(outputMapper, " ");
```

Comment:
what are you checking here?

Reply:
I forgot to delete an unnecessary test to check if the tests are working correctly.